app/actors/curation_concerns/actors/apply_order_actor.rb

```diff
@@ -3,11 +3,29 @@ module Actors
     class ApplyOrderActor < AbstractActor
       def update(attributes)
         ordered_member_ids = attributes.delete(:ordered_member_ids)
+        sync_members(ordered_member_ids)
         apply_order(ordered_member_ids) && next_actor.update(attributes)
       end
 
       private
 
+        def sync_members(ordered_member_ids)
+          return true if ordered_member_ids.nil?
+          existing_members_ids = curation_concern.member_ids
+          (existing_members_ids - ordered_member_ids).each do |old_id|
+            work = ::ActiveFedora::Base.find(old_id)
+            curation_concern.ordered_members.delete(work)
+          end
+          curation_concern.save
```

Comment:
Does an integration spec pass if this actor just doesn't save anything? It didn't before, and it'd make things that much quicker.